src/modules/uac_redirect/rd_funcs.c

```diff
@@ -33,9 +33,9 @@
 
 
 extern sruid_t _redirect_sruid;
+extern default_q_value;
```

Comment:
This one is missing the type of the variable, expecting some compilers to at least throw a waring about, should be:

```
extern int default_q_value;
```

Reply:
Of course. Missed that. Fixed.